editoast/src/views/infra/routes.rs

```diff
@@ -121,6 +121,12 @@ struct RouteTrackRangesParams {
     routes: List<String>,
 }
 
+#[derive(Default, Debug, Serialize, Deserialize, PartialEq, ToSchema)]
+struct RoutesFromNodesPositions {
+    routes: Vec<String>,
+    available_node_positions: HashMap<String, Vec<String>>,
```

Comment:
We should use `HashSet` since we don't want duplicated values. It will also simplify the code bellow.

```suggestion
    available_node_positions: HashMap<String, HashSet<String>>,
```